web/src/lib/components/shared-components/update-panel.svelte

```diff
@@ -27,7 +27,7 @@
           });
         } else {
           notificationController.show({
-            message: `${assetUpdateCount} assets updated.\nPlease reload to apply changes`,
+            message: `${assetUpdateCount} assets updated.`,
```

Comment:
I think this change should be reverted.

Reply:
No you don't need to reload anymore :)

Reply:
I thought the consensus was to *not* use websockets...? @jrasm91 @alextran1502

Reply:
We agreed with @jrasm91 to update the front with websocket, it acctualy feel better for the user